packages/babel-preset/__fixtures__/unsupported-css-properties/code.ts

```diff
@@ -0,0 +1,100 @@
+import { makeStyles } from '@griffel/react';
+
+export const useStyles = makeStyles({
+  root: {
+    // @ts-expect-error unsupported css properties should be stripped
+    all: 'value',
```

Comment:
I suggest to trim this file and keep only common properties (`margin`, `padding`, `border`) as technically it's just integration test for `resolveStyles()`.

Reply:
it's a good call, I limited to what I think of as 'common' shorthands